### DISCUSSION TITLE
Multi-disc compilations with multiple different recording/album artists per disc

### DISCUSSION THREAD

**Discussion by cursiefen** [navidrome/navidrome#3219] [General]
Hello everybody,

I am facing the fowolling issue (and couldn't find an answer here): As far as I cann see, Navidrome breaks multi-disc compilations with multiple different recording or album artists respectively per disc, to be specific classical multi-disc editions of a composer with different performing artists per disc, into different albums for each recording/album artist. 

Is there a way to avoid this behaviour, for example by appropriate tagging? This would be nice not only for navidrome, but also with regard to clients (like Symfonium) that allow multi-disc albums.

**Comment by metalheim** [comment]
You can add the "compilation" tag (tcmp=1) and navidrome will cluster the album together into one, but will only show "various artists" for albumartist (because it is a multi-albumartist compilation).

Other than that, you cant do much at the moment. If you set different albumartists, then navidrome must assume that they are different albums. You can have different ARTIST and one single ALBUMARTIST though and navidrome will group them together.

**Comment by cursiefen** [comment]
Thank you!

I have set the compilation tag for all songs, but this doesn't change Navidrome's behaviour of breaking down one disc of a multi-disc compilation with different album/recording artists per disc into different albums, one for each recording/album artist.

Tagging composer als ALBUMARTIST and recording or performing artist respectively as ARTIST simply doesn't feel right to me, and it doesn't comply with the Musicbrainz Classical Style Guide, as far as I can tell. 

So, I guess, I'm stuck.

**Reply by dweymouth** [reply]
Without adding the logic to use something like MUSICBRAINZ_ALBUM_ID to group albums (or if that tag isn't present), the best thing Navidrome can assume is the current behavior - where the ALBUMARTIST tag plus the ALBUM name and year determine a unique album

**Reply by cursiefen** [reply]
Thank you. 

Is there any reason not to post this (adding MUSICBRAINZ_ALBUM_ID to group albums ) as a feature request? Frankly, I'm not familiar with the customs of software projects.

**Reply by deluan** [reply]
> adding MUSICBRAINZ_ALBUM_ID to group albums

This is being done in #2709 and there is a fork that implements this right now, #1363, maintained by @vs49688) If you need it now, you can give it a try. Or else wait for #2709

> Frankly, I'm not familiar with the customs of software projects.

Just search for it in Issues and Discussions. (or even [Google](https://www.google.com/search?q=site%3Agithub.com%2Fnavidrome%2Fnavidrome+musicbrainz_album_id&oq=site%3Agithub.com%2Fnavidrome%2Fnavidrome+musicbrainz_album_id&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIGCAEQRRg60gEJMTc5NzJqMGoxqAIAsAIA&sourceid=chrome&ie=UTF-8)). If you can't find anything related to what you want, just post in our ["Ideas"](https://github.com/navidrome/navidrome/discussions/categories/ideas) forum, or reach out in Discord.

**Reply by cursiefen** [reply]
Thank you, I'll wait for #2709.

**Comment by wie-was** [comment]
Not sure if this is the same exact use case, but after some experimentation with tagging, I got my Alfred Schnittke 6-CD edition to be displayed quite nicely: 

![image](https://github.com/user-attachments/assets/d6f21091-a0f8-4f76-ae1f-3b78213e04c3)

The specifics:

discnumber = numer of disc, eg. "2"
tsst = name of disc, eg. "Symphonie Nr. 2, 'St. Florian'"
album artist = "Various Artists"
Album = "Alfred Schnittke. The Ten Symphonies"

**Reply by cursiefen** [reply]
Yes, thank you, that's exactly the same use case, and one can certainly do it like that. 

But still, this doesn't feel right to me, taking the risk of simply being on the wrong track here: In your case, with regard to some of the discs of the collection, the composer's name can only be found in the "album title" tag, and maybe, I reckon, in the "composer" tag, but neither in the "artist" tag nor the "album artist" tag. So in case I want to search the database for music by "Alfred Schnittke", I would have to search album titles for that string. That seems a bit odd, doesn't it?
